### PR TITLE
Update jib-gradle-plugin/CHANGELOG for the new option: `configurationName`

### PR DESCRIPTION
**Pull request by mpeddada1**
For #3034

### CODE REVIEW

File: jib-gradle-plugin/README.md [hunk 1]
```diff
@@ -193,6 +193,7 @@ Field | Type | Default | Description
...
 `containerizingMode` | `String` | `exploded` | If set to `packaged`, puts the JAR artifact built by the Gradle Java plugin into the final image. If set to `exploded` (default), containerizes individual `.class` files and resources files.
 `allowInsecureRegistries` | `boolean` | `false` | If set to true, Jib ignores HTTPS certificate errors and may fall back to HTTP as a last resort. Leaving this parameter set to `false` is strongly recommended, since HTTP communication is unencrypted and visible to others on the network, and insecure HTTPS is no better than plain HTTP. [If accessing a registry with a self-signed certificate, adding the certificate to your Java runtime's trusted keys](https://github.com/GoogleContainerTools/jib/tree/master/docs/self_sign_cert.md) may be an alternative to enabling this option.
+`configurationName` | `String` | `runtimeClasspath` | Configures the name of the [Gradle Configuration](https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.Configuration.html) to use. It can be used to be selective about the dependencies and artifacts added to the container.
```
Comment (chanseokoh) [inline jib-gradle-plugin/README.md:196]: This change should be in a separate PR so that we can merge it after making the next release. 

```suggestion
`configurationName` | `String` | `runtimeClasspath` | Configures the name of the [Gradle Configuration](https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.ConfigurationContainer.html) to use. It can be used to be selective about the dependencies and artifacts added to the container.
```
The new link seems more user-friendly.

File: jib-gradle-plugin/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 
+- Added an option `configurationName` to specify the name of the [Gradle Configuration](https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.Configuration.html) to use. This change introduces lazy evaluation by exposing the Gradle Property through a getter, which is different from the traditional route of using Gradle Provider and Property. ([#3034](https://github.com/GoogleContainerTools/jib/pull/3034))
+  ```gradle
+  jib {
+    configurationName.set("myconfig")
+  }
+  ```
```
Comment (chanseokoh) [inline jib-gradle-plugin/CHANGELOG.md:13]: I'm pretty sure you can do `configurationName = 'myconfig'`, but you can double-check. How about this?

```suggestion
- Added an option `configurationName` to specify the name of the [Gradle Configuration](https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.Configuration.html
) to use. The option can be lazily configured, for example, using Gradle `Provider` or `Property`. ([#3034](https://github.com/GoogleContainerTools/jib/pull/3034))
  ```gradle
  jib {
    configurationName.set = 'myconfig'
  }
  ```
```

File: jib-gradle-plugin/README.md (hunk 1 above)
Comment (mpeddada1) [reply jib-gradle-plugin/README.md]: Good catch. I will move this change out of this PR.

File: jib-gradle-plugin/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 
+- Added an option `configurationName` to specify the name of the [Gradle Configuration](https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.ConfigurationContainer.html) to use. The option can be lazily configured, for example, using Gradle `Provider` or `Property`. ([#3034](https://github.com/GoogleContainerTools/jib/pull/3034))
+```gradle
+    jib {
+      configurationName.set = 'myconfig'
```
Comment (chanseokoh) [inline jib-gradle-plugin/CHANGELOG.md:11]: Ah, my bad. I think it's just
```suggestion
      configurationName = 'myconfig'
```
Right? (I think it worked in this way.)

**Review by chanseokoh — APPROVED** [review]
I think the syntax is invalid, but other than that, LGTM.

File: jib-gradle-plugin/CHANGELOG.md (hunk 3 above)
Comment (mpeddada1) [reply jib-gradle-plugin/CHANGELOG.md]: My bad, thank you. Just did a quick test and `configurationName = 'myConfig'` works.

**Review by chanseokoh — APPROVED** [review]
